### PR TITLE
A test for the enrollment roster api.

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

**Comment by dcadams** [comment]
retest this please

**Comment by dcadams** [comment]
@stvstnfrd

File: common/djangoapps/enrollment/tests/test_data.py [hunk 1]
```diff
@@ -256,3 +257,58 @@ def test_update_for_non_existent_user(self):
...
+        self.staff.save()
+        UserProfileFactory.create(user=self.staff, name=self.STAFF_NAME)
+
+        CourseEnrollment.enroll(self.student, self.course_key, mode="honor")
+        CourseEnrollment.enroll(self.staff, self.course_key, mode="honor")
+        CourseStaffRole(self.course_key).add_users(self.staff)
+
+    def test_roster(self):
+
+        roster = data.get_roster(self.course_key.to_deprecated_string())
```
Comment (stvstnfrd) [inline common/djangoapps/enrollment/tests/test_data.py]: New code should not call `to_deprecated_string`;
isn't the new syntax just `unicode(self.course_key)`?

**Comment by stvstnfrd** [comment]
@dcadams After the `to_deprecated_string` change, I'm +1 here.

**Comment by dcadams** [comment]
fyi: @jbassen

**Pull request merged**